testdata/p4_16_samples/issue561-3-bmv2.stf

```diff
@@ -0,0 +1,7 @@
+packet 0 00 00 00 00
+packet 0 01 00 00 00
+packet 0 02 00 00 00
+
+expect 0 00 FF FF
+expect 0 01 FF
```

Comment:
I do not see how the above two expected packet output lines can be correct.

I would have expected, I think:
```
expect 0 00 FF FF 00 00
expect 0 01 FF 00
```

Reply:
I think that STF specifies a prefix of the packet. So my file implies yours. But yours is more precise.